tracing/README.md

```diff
@@ -0,0 +1,21 @@
+# Tracing
```

Comment:
Would `docs/tracing` be a better home for these?  Or at least the `README` as `docs/open-telemetry.md`?  I'm concerned that this may get overlooked as just another module.  Weakly held opinion.